src/UKHO.ERPFacade.Common/IO/Azure/AzureQueueMessaging.cs

```diff
@@ -0,0 +1,60 @@
+﻿using System.Diagnostics.CodeAnalysis;
+using Microsoft.Extensions.Options;
+using UKHO.ERPFacade.Common.Configuration;
+using Newtonsoft.Json.Linq;
+using Newtonsoft.Json;
+using UKHO.ERPFacade.Common.Models;
+using Azure.Storage.Queues;
+using UKHO.ERPFacade.Common.Models.QueueEntities;
+
+namespace UKHO.ERPFacade.Common.IO.Azure
+{
+    [ExcludeFromCodeCoverage]
+    public class AzureQueueMessaging : IAzureQueueMessaging
+    {
+        private readonly IOptions<AzureStorageConfiguration> _azureStorageConfig;
+        private const string RecordOfSaleQueueName = "recordofsaleevents";
```

Comment:
can we keep queue name in appsettings.json file please?

Reply:
Not required for now